net-ftp/ftp/ftp-0.17.34.0.2.5.1.ebuild

```diff
@@ -56,6 +56,9 @@ src_configure() {
 	append-lfs-flags
 	tc-export CC
 
+	# bug #943875
+        append-cflags " -std=gnu17"
```

Comment:
The indentation is wrong here (ebuilds use tabs). You also don't need the whitespace in the quotes. The quoting is optional (not wrong) but I wouldn't bother with it here.

Reply:
Trying to fix it now. I may take this PR draft down.

Reply:
Just leave it as a draft and force-push to fix it, then mark it as ready to review when it is ready.